app/main.js

```diff
@@ -1 +1,8 @@
 chrome.power.requestKeepAwake("display");
+
+window.addEventListener('keydown', event => {
```

Comment:
Technically if you want to replicate normal reload functionality this should be keyup